doc/async_operations.md

```diff
@@ -1,5 +1,24 @@
 # Asynchronous operations
 
-You are reading a draft of the next documentation and it's in continuos update so
-if you don't find what you need please refer to:
-[C++ wrapper classes for the ABI-stable C APIs for Node.js](https://nodejs.github.io/node-addon-api/)
\ No newline at end of file
+Node.js native add-ons often need to execute long running tasks and to avoid
+blocking the **event loop** they have to run them asynchronously from the
+**event loop**.
+In the Node.js model of execution there are **two threads**, the first is the
```

Comment:
Are there actually two threads?  There's the main event loop thread that's constant and worker threads as needed.

Reply:
Maybe I can rewrite the sentence like this:
In the Node.js model of execution the **event loop** thread represents the thread where JavaScript code is executing.

Reply:
I think that's better.